src/main/java/seedu/bookmark/model/Library.java

```diff
@@ -15,6 +15,7 @@
  */
 public class Library implements ReadOnlyLibrary {
 
+    public static final int MAX_BOOK_CAPACITY = 100;
```

Comment:
I notice that max book capacity was repeated in Model.java as well, perhaps this value could be imported as a static variable from another file?

Reply:
Or maybe this can be removed since I don't think its being used anywhere

Reply:
Nice catch, thanks!